### PR TITLE
Improving robot

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This fixes delete tests. It removes sleep and set timeout now to 60 sec.

### CODE REVIEW

File: components/tests/ui/testcases/web/delete_test.txt [hunk 1]
```diff
@@ -13,12 +13,19 @@ Suite Teardown      Close all browsers
 
 ${thumbnailsXpath}      //ul[@id='dataIcons']//div[contains(@class, 'image')]
 
+*** Keywords ***
+
+Cleal activity
```
Comment (sbesson) [inline components/tests/ui/testcases/web/delete_test.txt]: Clear ?

**Comment by aleksandra-tarkowska** [comment]
Job is green https://ci.openmicroscopy.org/view/Failing/job/OMERO-5.1-merge-robotframework/345/

**Comment by aleksandra-tarkowska** [comment]
--rebased-to #4114

File: components/tests/ui/resources/robot.template [hunk 2]
```diff
@@ -20,7 +20,7 @@ ${SERVER_ID}            1
 ${BROWSER}              Firefox
 ${DELAY}                0
 ${WAIT}                 50
-${TIMEOUT}              1
+${TIMEOUT}              60
```
Comment (sbesson) [inline components/tests/ui/resources/robot.template]: Just to be clear, this means the timeout of all tests - not just Delete tests - is extended to 1 minute, correct?

Comment (aleksandra-tarkowska) [inline components/tests/ui/resources/robot.template]: Timeout for all loops ``Wait until keyword succeed`` is meant to be 1 min. I accidentally put 1s rather then 1min=60sec.

Comment (aleksandra-tarkowska) [inline components/tests/ui/resources/robot.template]: It was reported by @will-moore and fixed in develop in https://github.com/will-moore/openmicroscopy/commit/4daa4f9b23dc7cf274c6c99ec9f473a8fe252af8

**Comment by sbesson** [comment]
Note while running additional runs of the robotframework job, the delete tests failed in the last 2 runs (https://ci.openmicroscopy.org/view/Failing/job/OMERO-5.1-merge-robotframework/349/robot/Web%20&%20Web/Web_1/Delete%20Test/).

File: components/tests/ui/testcases/web/delete_test.txt [hunk 3]
```diff
@@ -74,19 +80,20 @@ Test Delete Images in Dataset
...
     ${delThumbCount}=                       Evaluate   ${thumbCount} - 1
-    Wait Until Keyword Succeeds             1   0.1   Xpath Should Match X Times   ${thumbnailsXpath}   ${delThumbCount}
-    # ...Need to check that centre panel doesn't reload and show image during delete: #12866
-    Sleep                                   5
+    Wait Until Keyword Succeeds     ${TIMEOUT}      ${INTERVAL}   Xpath Should Match X Times   ${thumbnailsXpath}   ${delThumbCount}
+    # Wait for activities to show job done, then refresh tree...
+    Wait Until Keyword Succeeds     ${TIMEOUT}   ${INTERVAL}   Page Should Contain Element     xpath=//span[@id='jobstatus'][contains(text(),'1')]
+    Click Element                           refreshButton
+    Wait Until Page Contains Element        xpath=//li[@rel='experimenter']//a[contains(@class, 'jstree-clicked')]  ${WAIT}
+    Wait Until Page Contains Element        id=dataIcons    ${WAIT}
```
Comment (will-moore) [inline components/tests/ui/testcases/web/delete_test.txt]: See https://github.com/openmicroscopy/openmicroscopy/pull/3792 and ticket (12866) to see if the bug-fix (first commit of the PR) is still caught by this test.

**Comment by sbesson** [comment]
From https://ci.openmicroscopy.org/view/Failing/job/OMERO-5.1-merge-robotframework/354/robot/Web%20&%20Web/Web/Delete%20Test/ and https://ci.openmicroscopy.org/view/Failing/job/OMERO-5.1-merge-robotframework/354/robot/Web%20&%20Web/Web/Delete%20Test/, merging.

**Comment by sbesson** [comment]
--rebased-to #3995 cherry-pick

**Pull request merged**